test/1_stdlib/UIKit.swift

```diff
@@ -1,77 +1,86 @@
-// RUN: %target-run-simple-swift | FileCheck %s
+// RUN: %target-run-stdlib-swift
 // REQUIRES: executable_test
 // REQUIRES: OS=ios
 
 import UIKit
+import StdlibUnittest
 
-func printOrientation(o: UIDeviceOrientation) {
-  print("\(o.isPortrait) \(UIDeviceOrientationIsPortrait(o)), ", terminator: "")
-  print("\(o.isLandscape) \(UIDeviceOrientationIsLandscape(o)), ", terminator: "")
-  print("\(o.isFlat), ", terminator: "")
-  print("\(o.isValidInterfaceOrientation) \(UIDeviceOrientationIsValidInterfaceOrientation(o))")
+let UIKitTests = TestSuite("UIKit")
+
+private func printOrientation(o: UIDeviceOrientation) {
+  var s = "\(o.isPortrait) \(UIDeviceOrientationIsPortrait(o)), "
+  s += "\(o.isLandscape) \(UIDeviceOrientationIsLandscape(o)), "
+  s += "\(o.isFlat), "
+  s += "\(o.isValidInterfaceOrientation) \(UIDeviceOrientationIsValidInterfaceOrientation(o))"
+  return s
+}
+
+UIKitTests.test("Device orientations") {
+  var s = printOrientation(.unknown)
+  expectEqual("false false, false false, false, false false", s)
```

Comment:
Why not `expectEqual(..., printOrientation(...))` and avoid the mutable variable?